.github/workflows/conda.yml

```diff
@@ -55,12 +56,14 @@ jobs:
 
           # distributed pre-release build
           conda mambabuild continuous_integration/recipes/distributed \
+                           --channel conda-forge \
```

Comment:
Was this (and the addition the `conda mambabuild` command below) required to get `versioneer` working as a host dependency? Looking at other build runs, it seems like our default channel should be conda-forge (assuming this is getting picked up from the Mambaforge installation in this workflow), so would assume we wouldn't need this

Reply:
Yes the build failed without this

Reply:
Is there a link to the failing build log?